doc/source/code_editors.rst

```diff
@@ -1,3 +1,35 @@
 ============
 Code Editors
 ============
+===================
+IDE Recommendations
+===================
+-NotepadPP
+	NotepadPP version 8.2 and the addon “reStructuredText_NPP”
+	Use the following link to GitHub for further instructions 
+	https://github.com/steenhulthin/reStructuredText_NPP
+
+-Vim 
+	Vim is a markdown editor which can be utilized for RST. For less experienced Users it would be better to use Visual Studio Code. VS Code is more user friendly in UI Design and overall experience
+	For Vim the plugin “RIV” is a good choice RIV has good support for RST. 
+	https://github.com/steenhulthin/reStructuredText_NPP
+
+-Nano
+	Next IDE is also targeted at more experienced users that aren´t interested in nice UI Designs. 
+	But I wouldn´t recommend using Nano because there are no RST Plugins for Nano
+
+-Atom
+	Atom greets you with the promise being “a hackable text editor for the 21st Century” 
+	Customization and Packages can be installed trough the integrated App store. 
+	To use RST in Atom you can install “rst-preview” 
+	https://atom.io/packages/rst-preview
```

Comment:
Please explain here explicitly that this can be only used as a preview, but can not be used as wysiwyg